### PR TITLE
Documentation for redirectPage function

### PR DESCRIPTION
**Pull request by bwallace-ch**
Changes:
- Adding documentation for redirectPage function in referrerUtils.ts

### CODE REVIEW

File: src/lib/utils/referrerUtils.ts [hunk 1]
```diff
@@ -1,5 +1,17 @@
 import logger from "../Logger";
 
+/**
+ *
```
Comment (mmodzelewski-ch) [inline src/lib/utils/referrerUtils.ts:4]: It would be good to add here some general information what this functions is for.

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/lib/utils/referrerUtils.ts [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+ * @param referrer - redirect url - the specific url prior to the url the user is currently on
```
Comment (mmodzelewski-ch) [inline src/lib/utils/referrerUtils.ts:5]: I would rather say something that informs that this is the url from which the user arrive at the current url, or something like that.

File: src/lib/utils/referrerUtils.ts [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+ * @param hrefA - expected previous page url - compared with redirect parameter to confirm if user has travelled from the page prior to the current page in the journey
```
Comment (mmodzelewski-ch) [inline src/lib/utils/referrerUtils.ts:6]: Do you mean "referrer parameter", not "redirect parameter"?

File: src/lib/utils/referrerUtils.ts [hunk 4]
```diff
@@ -1,5 +1,17 @@
...
+ * @param hrefB - expected current page url - compared with redirect parameter to account for whether user has changed language on current page, changing the url
```
Comment (mmodzelewski-ch) [inline src/lib/utils/referrerUtils.ts:7]: as above

File: src/lib/utils/referrerUtils.ts [hunk 5]
```diff
@@ -1,5 +1,17 @@
...
+ * @param pageIndicator - for when the authentication system causes referrer paramater to be 'undefined'
+ *                      - if true, page is not redirected as user has travelled from a valid page
+ * @param hrefC - expected next page - compared with redirect parameter to account for whether a user has clicked 'back' to go to the previous page
```
Comment (mmodzelewski-ch) [inline src/lib/utils/referrerUtils.ts:10]: as above

**Review by mmodzelewski-ch — CHANGES_REQUESTED** [review]
See the individual comments

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/lib/utils/referrerUtils.ts [hunk 6]
```diff
@@ -1,5 +1,20 @@
...
+ *
+ * @param referrer - redirect url - the url from which the user has arrived from to get to the current url
+ *                 - is undefined if a user has bookmarked the page they wish to access
+ * @param hrefA - expected previous page url - compared with referrer parameter to confirm if user has travelled from the page prior to the current page in the journey
+ * @param hrefB - expected current page url - compared with referrer parameter to account for whether user has changed language on current page, changing the url
+ * @param pageIndicator - for when the authentication system causes referrer paramater to be 'undefined'
+ *                      - if true, page is not redirected as user has travelled from a valid page
+ * @param hrefC - expected next page - compared with referrer parameter to account for whether a user has clicked 'back' to go to the previous page
+ *              - optional as not all pages have a page that follows them
+ * @returns
```
Comment (mmodzelewski-ch) [inline src/lib/utils/referrerUtils.ts:15]: State what the functions returns. Something like "true if yada yada yada, false otherwise".

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by mmodzelewski-ch — APPROVED** [review]

**Pull request merged**